apps/explorer/src/pages/transaction-result/TransactionCard.tsx

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ChevronDown12, ChevronUp16 } from '@mysten/icons';
+import clsx from 'clsx';
+import { type ReactNode, useCallback, useEffect, useState } from 'react';
+
+import { Button } from '~/ui/Button';
+import { Card, type CardProps } from '~/ui/Card';
+import { Divider } from '~/ui/Divider';
+import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
+
+type Size = 'md' | 'sm';
+
+export function useTransactionCardControl(
```

Comment:
This UI feels a little unintuitive to me. Took me a while to figure out the controls. Why not just have individual expandable + hide after some # of inputs + make inputs tab collapsable. I think that'd be a better UX here, instead of having buttons that control each of the individual expand states of the inputs. cc @mystie711

Reply:
Spoke with Jordan. I like the logic simplification.

@plam-ml tet's update the logic to,
- show up to 5 inputs expanded by default
- "Show all X Inputs" shows all the rest expanded by default
-- when expanded the input section should show a maximum of 8 inputs in the scrolling container. This should help with smaller size screens and single column layouts

Reply:
@mystie711 this is not easy to do since `maximum of 8 inputs` can have very variable heights (some input has 3 rows vs 5 or more). Ideally, we should not try to compute the size of each row and set the maximum heights accordingly. Let's sync later today on this.

Reply:
Got it. 
- let's keep all inputs collapsed by default
- let's keep the unexpanded views for the Inputs when we click "Show all X Inputs"
- use fixed height of 10 unexpanded Inputs for the scroll area?

Reply:
we could change this 10 number to something else in future if it makes sense based on use cases we see.

Reply:
@plam-ml You could probably modify `ExpandableList` to handle this use-case

Reply:
We should have consistent behavior, so if `ExpandableList` doesn't work we've done something wrong here.

Reply:
You can also have an initial expanded state beyond the first certain number of elements to satisfy this requirement if we want.